siswrap/wrapper_services.py

```diff
@@ -190,11 +215,18 @@ def run(self):
     def url_to_type(url):
         """ Helper method to see which wrapper object might belong to which URL.
         """
-        # TODO: Take out the correct part of the URL instead.
-        if Wrapper.QC_TYPE in url:
+
+        url_pattern = "\/api/\d.\d\/(\w+)\/"
+        first_match = re.search(url_pattern, url).group(1)
+
+        if first_match == Wrapper.QC_TYPE:
             return Wrapper.QC_TYPE
-        elif Wrapper.REPORT_TYPE in url:
+        elif first_match == Wrapper.REPORT_TYPE:
             return Wrapper.REPORT_TYPE
+        elif first_match == Wrapper.AEACUS_STATS_TYPE:
+            return Wrapper.AEACUS_STATS_TYPE
+        elif first_match == Wrapper.AEACUS_REPORTS_TYPE:
+            return Wrapper.AEACUS_REPORTS_TYPE
         else:
             raise RuntimeError("Unknown wrapper runner requested: {0}".
                                format(url))
```

Comment:
So many wrapper types back and forth in both `url_to_type()` and `new_wrapper()`. Guess this could be restructured more cleanly in the future.. some happy day..